pkg/siftool/add.go

```diff
@@ -284,7 +301,7 @@ func (c *command) getAdd() *cobra.Command {
 		}
 		defer f.Close()
 
-		opts, err := getOptions(dt, cmd.Flags())
+		opts, err := getOptions(dt, cmd.Flags(), f)
```

Comment:
Nit: this works, but the separation of concerns isn't clean. Looking at `getAdd`, it isn't obvious whether the cursor in `f` is positioned properly to use as an `io.Reader` that will read the entire file when `getOptions` returns. Looking at `getOptions`, it isn't obvious that `f` _must_ be reset for its caller(s) to function properly.

To make it clearer, how about we:

- Use an `io.Reader` instead of `*os.File` to pass the file to `getOptions`. This makes it explicit that `getOptions` will read the file, and that it isn't responsible for anything other than that.
- Move the `f.Seek` to the `func cmd.RunE`, between the `getOptions` and `c.app.Add` calls. This makes it clear that `f` is good to go for the call to `c.app.Add`.

Reply:
Think this one was hidden under "resolved" in the last review, my apologies.